### PR TITLE
Port latest changes from the Private preview branch to Samples 

### PR DESCRIPTION
**Pull request by atchutbarli**
port changes

## Purpose
Port the changes from private preview branch to the k8s branch
* ...

## Does this introduce a breaking change?
NO

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Documentation content changes

```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: samples/k8s-offer-azure-vote/azure-vote/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v2
+description: A Helm chart for Kubernetes
```
Comment (RamyasreeChakka) [inline samples/k8s-offer-azure-vote/azure-vote/Chart.yaml:2]: >A Helm chart for Kubernetes [](http://example.com/codeflow?start=13&length=27)

A Helm chart for Azure Vote application for Kubernetes

File: samples/k8s-offer-azure-vote/azure-vote/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# azure-vote
+
+The Azure Vote application is a simple sample used throughout the Azure Container Service (AKS) documentation. The application consists of two pods, one running a flask web form, and the second a redis instance for temporary data storage.
```
Comment (RamyasreeChakka) [inline samples/k8s-offer-azure-vote/azure-vote/README.md:3]: >Azure Container Service (AKS) documentation [](http://example.com/codeflow?start=66&length=43)

Azure Marketplace Azure Container Kubernetes Apps offer documentation

File: samples/k8s-offer-azure-vote/azure-vote/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+name: azure-vote
```
Comment (RamyasreeChakka) [inline samples/k8s-offer-azure-vote/azure-vote/Chart.yaml:3]: >azure-vote [](http://example.com/codeflow?start=6&length=10)

k8s-offer-azure-vote

File: samples/k8s-offer-azure-vote/azure-vote/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Add the Azure Samples chart repository.
+
+```
+helm repo add azure-samples https://azure-samples.github.io/helm-charts/
+```
+
+Install the chart.
+
+```
+helm install azure-samples/azure-vote
```
Comment (RamyasreeChakka) [inline samples/k8s-offer-azure-vote/azure-vote/README.md:16]: >azure-vote [](http://example.com/codeflow?start=27&length=10)

k8s-offer-azure-vote

File: samples/k8s-offer-azure-vote/azure-vote/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+| title | Azure vote app title. | Azure Vote App |
+| value1 | Value for first vote control / value | Cats |
+| value2 | Value for second vote control / value | Dogs |
+| serviceName | Name for Kubernetes service. | aks-helloworld |
+| serviceType | Type for Kubernetes service. | ClusterIP |
+
+## Examples
+
+```
+helm install azure-samples/azure-vote
```
Comment (RamyasreeChakka) [inline samples/k8s-offer-azure-vote/azure-vote/README.md:35]: >azure-vote [](http://example.com/codeflow?start=27&length=10)

k8s-offer-azure-vote

File: samples/k8s-offer-azure-vote/azure-vote/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+![azure-vote cats and dogs](images/vote1.png)
```
Comment (RamyasreeChakka) [inline samples/k8s-offer-azure-vote/azure-vote/README.md:38]: >vote1.png [](http://example.com/codeflow?start=35&length=9)

vote1.png and vote2.png Images are missing in this checkin

File: samples/k8s-offer-azure-vote/azure-vote/README.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+```
+helm install azure-samples/azure-vote --set title="Winter Sports" --set value1=Ski --set value2=Snowboard
```
Comment (RamyasreeChakka) [inline samples/k8s-offer-azure-vote/azure-vote/README.md:41]: >azure-vote [](http://example.com/codeflow?start=27&length=10)

k8s-offer-azure-vote

File: samples/k8s-offer-azure-vote/azure-vote/templates/NOTES.txt [hunk 8]
```diff
@@ -0,0 +1,10 @@
+
+The Azure Vote application has been started on your Kubernetes cluster.
+
+Title: {{ .Values.title }}
+Vote 1 value: {{ .Values.value1 }}
+Vote 2 value: {{ .Values.value2 }}
+
+The externally accessible IP address can take a minute or so to provision. Run the following command to monitor the provisioning status. Once an External IP address has been provisioned, brows to this IP address to access the Azure Vote application.
```
Comment (RamyasreeChakka) [inline samples/k8s-offer-azure-vote/azure-vote/templates/NOTES.txt:8]: >brows [](http://example.com/codeflow?start=187&length=5)

browse

**Review by RamyasreeChakka — APPROVED** [review]
:shipit:

File: samples/k8s-offer-azure-vote/azure-vote/README.md (hunk 6 above)
Comment (atchutbarli) [reply samples/k8s-offer-azure-vote/azure-vote/README.md:38]: i see them in the images directory

**Pull request merged**